src/it/resources/create_mails_and_mailboxes.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+java -jar /root/james-cli.jar -h 127.0.0.1 -p 9999 CreateMailbox '#private' bart@simpson.cartoon rmbx0
```

Comment:
I think that for inbox home loading, you don't need to select any folder

Reply:
It will be needed for `Switching to another mailbox shoud take less than 1 second` BTW, we can keep the code if it doesn't take too long to make it settle, otherwise, remove it for now and keep it in a branch for future need.

Reply:
I updated the scenario, and stored the random mailboxes code here :
https://github.com/remk/james-gatling/tree/inbox_home_scenario_with_random_mailboxes
For future reference